core/org.eclipse.cdt.core/.settings/.api_filters

```diff
@@ -166,6 +166,22 @@
             </message_arguments>
         </filter>
     </resource>
+    <resource path="src/org/eclipse/cdt/core/build/ICBuildConfiguration.java" type="org.eclipse.cdt.core.build.ICBuildConfiguration">
```

Comment:
With CDT 11 we can just make this a new major version of the bundle and not have to have this API filter.

WDYT?